libs/blocks/tabs/tabs.css

```diff
@@ -323,6 +343,66 @@
   border: none;
 }
 
+.tabs.radio {
+  background: none;
+}
+
+.tabs.radio .tab-content {
+  border: none;
+}
+
+.tabs.radio .tab-paddles {
+  display: none;
+}
+
+.tabs.radio div[role="tablist"] {
+  margin: 0 auto;
+  width: var(--grid-container-width);
+}
+
+.tabs.radio div[role="tablist"] .tab-list-container {
+  border: 1px solid #d9d9d9;
```

Comment:
for consistency, should this be the `--tabs-border-color: #d8d8d8;` instead, may also help inherit dark theming?